shared/utility/motion/WalkEngine.hpp

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
and then get rid of this and the .cpp in the motion folder, in favor of the skill folder